shared/src/store/action-reducers/patient.ts

```diff
@@ -108,7 +109,21 @@ export namespace PatientActionReducers {
                 );
             }
             draftState.patients[patient.id] = cloneDeepMutable(patient);
-            calculateTreatments(draftState);
+
+            if (patient.position === undefined) {
+                throw new ReducerError(
+                    `Patient with id ${patient.id} can't be added without a position`
+                );
+            }
```

Comment:
Also a breaking change. Previously, one could add a patient that was already in a vehicle.